toolchain/parse/typed_nodes.h

```diff
@@ -859,6 +857,16 @@ struct ImplForall {
   ImplicitParamListId params;
 };
 
+// `as` with no type before it
```

Comment:
```suggestion
// `as` with no type before it.
```

Similarly, this feels more like a sentence with the additional words, so punctuation might make sense.

https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar

Reply:
This is a noun phrase not a sentence, and adding a period would be inconsistent with the parallel comments